src/Components/SortControl/__tests__/SortControl.test.js

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react';
+import { SortControl } from '../SortControl';
+import userEvent from '@testing-library/user-event';
+
+describe('SortControl', () => {
+  test('renders all options', () => {
+    render(<SortControl selectedOption={''} onSelect={jest.fn()} />);
+
+    expect(screen.getAllByRole('option')).toHaveLength(2);
+    expect(screen.getByRole('option', { name: 'Title' })).toBeInTheDocument();
+    expect(screen.getByRole('option', { name: 'Release Date' })).toBeInTheDocument();
```

Comment:
And here as well you can check for `label` of `sortOptions`'s entries instead of hard coding "Title" and "Release Date".